shogun-lib/src/main/java/de/terrestris/shogun/lib/model/jsonb/application/DefaultToolConfig.java

```diff
@@ -0,0 +1,37 @@
+/* SHOGun, https://terrestris.github.io/shogun/
+ *
+ * Copyright © 2022-present terrestris GmbH & Co. KG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.terrestris.shogun.lib.model.jsonb.application;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.io.Serializable;
+
+@Data
+@ToString
+@EqualsAndHashCode
+public class DefaultToolConfig implements Serializable {
+
+    @Schema(
+        description = "The visibility of the tool.",
+        example = "true"
+    )
+    private Boolean visible;
+
+}
```

Comment:
Missing new line. This would normally added automatically. Have you installed the `editorconfig` plugin to your IDE?

Reply:
I have now :)